### PR TITLE
fix(aws-ecs): remove temporary workaround for long arn support

### PR DESCRIPTION
**Pull request by piradeepk**
Remove the temporary workaround implemented to allow long arns support: [#2176](https://github.com/awslabs/aws-cdk/pull/2176)
* Removing the temporary workaround does not break anything for my account which has long arns enabled.

----

Please read the [contribution guidelines](https://github.com/awslabs/aws-cdk/blob/master/CONTRIBUTING.md) and follow the pull-request checklist.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-ecs/lib/base/base-service.ts [hunk 1]
```diff
@@ -73,21 +73,6 @@ export interface BaseServiceProps {
...
-   * enabled.
-   *
-   * Old ARN format: arn:aws:ecs:region:aws_account_id:service/service-name
-   * New ARN format: arn:aws:ecs:region:aws_account_id:service/cluster-name/service-name
-   *
-   * See: https://docs.aws.amazon.com/AmazonECS/latest/userguide/ecs-resource-ids.html
-   *
-   * @default false
-   */
-  readonly longArnEnabled?: boolean;
```
Comment (eladb) [inline packages/@aws-cdk/aws-ecs/lib/base/base-service.ts]: This is a breaking change. I suggest that you simply add a @deprecated tag to it and indicate that it doesn’t have any effect anymore.

Comment (eladb) [reply packages/@aws-cdk/aws-ecs/lib/base/base-service.ts]: Please apply the @deprecated tag and we will remove in the next major version.

Comment (pkandasamy91) [reply packages/@aws-cdk/aws-ecs/lib/base/base-service.ts]: Will do

**Comment by pkandasamy91** [comment]
Builds failing due to 'Some packages seem to have undergone breaking API changes. Please avoid.'

**Pull request merged**